fastestimator/network.py

```diff
@@ -454,36 +513,20 @@ def _detach_tensor(self, data: T) -> T:
         elif isinstance(data, torch.Tensor):
             return data.detach()
 
-    def transform(self, data: Dict[str, Any], mode: str, epoch: int = 1) -> Dict[str, Any]:
```

Comment:
we need to see the training speed before removing the `transform`

Reply:
transform is still there, and has nothing to do with training speed